src/components/FormInput.js

```diff
@@ -22,25 +23,43 @@ class FormInput extends React.Component {
 
   onInputChange(event) {
     let type=event.target.type;
-    let value=event.target.value;
+    let value=event.target.value; 
+
     if(value) {
       this.setState({ hasText: true })
     }
     else{
       this.setState({ hasText: false })
     }
-    this.validateField(type, value);
+    
+    this.validateField(type, value);   
   }
     
   validateField(type, value) {
-
     let errorMessage='';
     switch(type) {
-      case 'text':
-        if (!value.match(/^[a-zA-Z ]*$/)) {         
-          errorMessage='Only Alphabets allowed';
+      case 'text':       
+        if (value.length===1) {         
+          errorMessage='Too Short!';
+        }
+        if (value.length<1) {         
+          errorMessage= this.props.label +' is Required';
```

Comment:
Instead of `value.length < 1` you could use `!value.length` - This way you are are checking the "truthiness" of the value. This would take care of both `undefined` and `0`
```suggestion
        if (!value.length) {
          errorMessage = this.props.label +' is Required';
        }
        if (value.length === 1) {         
          errorMessage = 'Too Short!';
```

Also try to put if conditions in logical order
1. check if its 0
2. then its just 1 
3. ...
This is just for better readability of code